jobbergate-api/tests/conftest.py

```diff
@@ -64,24 +66,28 @@ async def synth_session():
     the dependency injection for the router code. Otherwise, changes made in the router's session would not
     be visible in the test code. Not that changes made in this synthesized session are always rolled back
     and never committed.
-    """
 
-    async with SessionLocal() as session:
+    NOTE:
+        Any router tests that interact with endpoints that use the database MUST use this fixture or the
```

Comment:
`pytestmark = pytest.mark.usefixtures("synth_session")` could be used on the files that test services and routers to make sure it is in place

Reply:
This was added to the router tests. I still want to leave this note for posterity.